sympy/simplify/cse_opts.py

```diff
@@ -37,7 +38,7 @@ def sub_post(e):
     for node in preorder_traversal(e):
         if isinstance(node, Mul) and \
             node.args[0] is S.One and node.args[1] is S.NegativeOne:
-            replacements.append((node, -Mul._from_args(node.args[2:])))
+            replacements.append((node, -Mul(*node.args[2:])))
```

Comment:
I think this one should also be evaluate=False. _from_args doesn't evaluate.

Reply:
Ah, stupid me, thanks.